tests/PitchTests.elm

```diff
@@ -0,0 +1,53 @@
+module PitchTests exposing (all)
+
+import Expect
+import Fuzz exposing (Fuzzer)
+import MusicTheory.Letter exposing (Letter(..))
+import MusicTheory.Octave as Octave
+import MusicTheory.Pitch as Pitch
+import MusicTheory.Pitch.Enharmonic as PitchEnharmonic
+import MusicTheory.PitchClass as PitchClass
+import MusicTheory.PitchClass.Enharmonic as PitchClassEnharmonic
+import Test exposing (..)
+import Util.OctaveFuzzer
+import Util.PitchClassFuzzer
+
+
+all : Test
+all =
+    describe "Pitch Tests"
+        [ fuzz2 Util.PitchClassFuzzer.pitchClass Util.OctaveFuzzer.octave "apply the same enharmonic transformation to pitch and to a pitch's inner pitch class should yield the same result" <|
+            \pitchClass octave ->
+                let
+                    enharmonicAppliedToPitch =
+                        pitchClass
+                            |> Pitch.fromPitchClass octave
+                            |> PitchEnharmonic.simple
+
+                    enharmonicAppliedToPitchClass =
+                        pitchClass
+                            |> PitchClassEnharmonic.simple
+                            |> Pitch.fromPitchClass octave
+                in
+                Expect.equal enharmonicAppliedToPitch enharmonicAppliedToPitchClass
```

Comment:
As far as I understand, this test asserts that Cb4 should be equal to B4. But enharmonic spelling of pitches doesn't work that way.

http://openmusictheory.com/pitches.html says:
"..a half step below C4 is C-flat4...it sounds the same as B3"

https://en.wikipedia.org/wiki/Scientific_pitch_notation says:
"enharmonic spellings can give rise to anomalies where C♭4 is a lower frequency than B♯3"

Reply:
I'm sorry— that last comment was rude. I know you agree that Cb4 and B3 are the same pitch. But I think I am frustrated because, after all our discussion, I don't understand why you think this characteristic is helpful.